### PR TITLE
ADXL345 Chain Anchor mod

### PR DESCRIPTION
**Pull request by padok**
* [X] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [X] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [X] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [X] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [X] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [X] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [X] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [X] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [X] Submitted STL files are oriented and scaled properly for printing.
  * [X] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
I modified the official "chain_anchor" part to act as a mount for the ADXL345 breakout boards.
I think it is a better solution than the other parts of similar function found here,  since after installation one can not see the ADXL345 board and thus it does not affect the overall appearance of the printer.

#### How was it tested? 
I've had it installed for a few months now, and since I unfortunately have to relocate my printer on a regular basis and recalibrate it every time, I can assure that the part serves the purpose for which it was designed.

#### Any background context you want to provide?
It's my first submission here, I hope I can deliver the expected quality.

#### Screenshots (if appropriate)
There are several screenshots included in the mod folder.

#### Further notes
/

### CODE REVIEW

File: printer_mods/padok/chain_anchor-ADXL345_mount/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Chain Anchor - ADXL345 mount Mod
+
+![Chain Anchor](../../padok/chain_anchor-ADXL345_mount/img/VORON2_v2.4_Assembly_-_Chain_Anchor_-_IGUS.png)
```
Comment (KiloQubit) [inline printer_mods/padok/chain_anchor-ADXL345_mount/README.md:3]: Please simplify all the image references, you don't need to go up two directories and then back down.  This image should be just

    Chain Anchor

File: printer_mods/padok/chain_anchor-ADXL345_mount/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+After that you can use this part as a replacement for the standard chain anchor.
+
+![Chain Anchor](../../padok/chain_anchor-ADXL345_mount/img/IMG_06.jpg)
+
+## BOM
+
+| #   | Part                                               |
+| --- | -------------------------------------------------- |
+| /   | Afterburner: Chain Anchor Parts                    |
+| 4   | M3 ruthex Threaded Insert                          |
```
Comment (KiloQubit) [inline printer_mods/padok/chain_anchor-ADXL345_mount/README.md:28]: What's a ruthex threaded insert?  Are these just standard inserts used everywhere else on a Voron?

File: printer_mods/padok/chain_anchor-ADXL345_mount/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+I printed the part on my printer using eSUN ABS+ Black/PeekGreen like all the other parts, but this one was unfortunately a little bit overextruded.
+After adding the threaded inserts, I sanded the plateau with some P180 sandpaper that I placed on a flat surface to ensure the best possible contact surface for the adapter board.
+
+![Chain Anchor](../../padok/chain_anchor-ADXL345_mount/img/IMG_01.jpg)
+
+After that you can use this part as a replacement for the standard chain anchor.
+
+![Chain Anchor](../../padok/chain_anchor-ADXL345_mount/img/IMG_06.jpg)
+
```
Comment (KiloQubit) [inline printer_mods/padok/chain_anchor-ADXL345_mount/README.md:22]: Can you also include IMG_04.jpg and IMG_05.jpg in this README?

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Your STL is not in a printable orientation.

Also please see README changes below.

File: printer_mods/padok/chain_anchor-ADXL345_mount/README.md (hunk 2 above)
Comment (padok) [reply printer_mods/padok/chain_anchor-ADXL345_mount/README.md:28]: Yes, these are the same as the regular ones, I just wanted to specify the brand I used so it's clear I mean the inserts that have the 45° external corners like these:

image

I didn't know how else to put it in short words.

**Comment by padok** [comment]
Thanks for your review @KiloQubit!

I hope I was able to accommodate your requests to your liking.

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Pull request merged**